edge_prediction_experiment/01_hetionet_recall_dag.py

```diff
@@ -282,19 +282,26 @@
 )
 print(g)
 
+fpr, tpr, roc_threshold = roc_curve(
```

Comment:
Seems like you're using similar code to this in a lot of places - could you put it in a function?

If it would take a lot of work to do so and you don't think the code will change much I'm fine with leaving it, up to you.

Reply:
This is a good point. I restructured the cells for things to make more sense.